src/main/java/com/dawsonsystems/session/MongoManager.java

```diff
@@ -35,8 +38,11 @@
 import java.util.UUID;
 import java.util.logging.Level;
 import java.util.logging.Logger;
+import org.apache.catalina.session.ManagerBase;
 
-public class MongoManager implements Manager, Lifecycle {
+import static com.dawsonsystems.session.ClientSSLFromPEMsUtility.sslContextFromPEMs;
+
+public class MongoManager extends ManagerBase implements Lifecycle {
```

Comment:
minor: `implements Lifecycle` isn't needed, as `ManagerBase` already implements it